app/models/user.js

```diff
@@ -1,41 +1,16 @@
 import Model, { belongsTo, attr } from '@ember-data/model';
-import { computed } from '@ember/object';
 
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-classes
-export default Model.extend({
-  firstName: attr('string'),
-  lastName: attr('string'),
-  emailLink: attr(),
-  phoneLink: attr(),
-  account: belongsTo('account'),
-  group: belongsTo('account-group', {
-    inverse: null,
-  }),
-  organization: belongsTo('organization'),
+export default class User extends Model {
+  @attr('string') firstName;
+  @attr('string') lastName;
+  @attr('email') email;
+  @attr('phone') phone;
 
-  email: computed('emailLink', {
-    get() {
-      return this.emailLink && this.emailLink.replace(/^mailto:/, '');
-    },
-    // eslint-disable-next-line no-unused-vars
-    set(key, value) {
-      this.set('emailLink', `mailto:${value}`);
-      return value;
-    },
-  }),
-  phone: computed('phoneLink', {
-    get() {
-      return this.phoneLink && this.phoneLink.replace(/^tel:/, '');
-    },
-    // eslint-disable-next-line no-unused-vars
-    set(key, value) {
-      this.set('phoneLink', `tel:${value.replace(/[/ .]/, '')}`);
-      return value;
-    },
-  }),
```

Comment:
same with tel, the removing of `tel:` no longer happens so the raw string will just be displayed
![image](https://user-images.githubusercontent.com/22245223/157633541-c393c648-207e-4797-892d-0662b9b2aabb.png)

Reply:
this is done by a transform, resolving